src/Czechia/Mews.Fiscalizations.Czechia.Tests/IntegrationTests/Basics.cs

```diff
@@ -30,7 +30,7 @@ public void TimeoutWorks()
             var certificate = CreateCertificate(Fixtures.Second);
             var record = CreateSimpleRecord(certificate, Fixtures.Second);
             var client = new EetClient(certificate, EetEnvironment.Playground, TimeSpan.FromMilliseconds(1));
-            Assert.ThrowsAsync<TaskCanceledException>(async () => await client.SendRevenueAsync(record));
+            Assert.ThrowsAsync<TaskCanceledException>(async () => await client.SendRevenueAsync(record).ConfigureAwait(continueOnCapturedContext: false));
```

Comment:
But here this should work. 
```suggestion
            Assert.ThrowsAsync<TaskCanceledException>(() => client.SendRevenueAsync(record));
```